rules/github_rules/github_repository_transfer.py

```diff
@@ -0,0 +1,47 @@
+from panther_base_helpers import github_alert_context
+
+
+def rule(event):
+    # Return True to match the log event and trigger an alert.
+    return (
+        event.get("action", "") == "repo.transfer"
```

Comment:
could condense this into event.get("action","") in ("repo.transfer", "repo.transfer_outgoing", "repo.transfer_start")

Reply:
condensed!